packages/web-components/src/components/masthead/__stories__/README.stories.mdx

```diff
@@ -41,6 +41,32 @@ import '@carbon/ibmdotcom-web-components/es/components/masthead/masthead-contain
 <dds-masthead-container platform="Lorem Ipsum"></dds-masthead-container>
 ```
 
+#### Setting Platform and URL
+
+```html
+<dds-masthead-container platform="Lorem Ipsum" platform-url="https://www.example.com"></dds-masthead-container>
+```
+
+#### Setting Platform URL Object (Optional)
+
+```javascript
+const urlObject = {
+  'en-US': {
+    url: 'https://www.example.com',
+  },
+  'fr-FR': {
+    url: 'https://www.example.com/fr-fr/sample',
+  },
+  'es-MX': {
+    url: 'https://www.example.com/sample/es-mx',
+  },
+};
+```
+
+```html
+<dds-masthead-container platform="Lorem Ipsum" platformObj={urlObject}></dds-masthead-container>
```

Comment:
if we're passing in as a property, I think it needs to have the `.` in front, so `.platformObj={urlObject}`